lib/internal/inspector/inspect_repl.js

```diff
@@ -707,7 +709,7 @@ function createRepl(inspector) {
     }
 
     if (typeof script !== 'string') {
-      throw new TypeError(`setBreakpoint() expects a string, got ${script}`);
+      throw new ERR_INVALID_ARG_TYPE('script', ['string'], script);
     }
```

Comment:
Can't we use a validator here instead?

Reply:
@Trott just making sure you've seen this nit, you could use `validateString` here.

Reply:
Sure, done.